### PR TITLE
Add ELB v3 loadbalancer

### PR DESCRIPTION
**Pull request by niuzhenguo**
Adds dedicated load balancer resource
**What this PR does / why we need it**:

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
Related to #1021 

**Special notes for your reviewer**:

**Release note**:

```release-note

```

## PR Checklist

* [x] Tests added/passed.
* [x] Documentation updated.
* [x] Schema updated.

## Acceptance Steps Performed

```
TF_ACC=1 go test ./huaweicloud/ -v -run=TestAccElbV3LoadBalancer -timeout 360m -parallel 4
=== RUN   TestAccElbV3LoadBalancer_basic
=== PAUSE TestAccElbV3LoadBalancer_basic
=== RUN   TestAccElbV3LoadBalancer_withEpsId
=== PAUSE TestAccElbV3LoadBalancer_withEpsId
=== CONT  TestAccElbV3LoadBalancer_basic
=== CONT  TestAccElbV3LoadBalancer_withEpsId
--- PASS: TestAccElbV3LoadBalancer_withEpsId (100.34s)
--- PASS: TestAccElbV3LoadBalancer_basic (189.90s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud       189.925s

```

### CODE REVIEW

File: huaweicloud/resource_huaweicloud_elb_loadbalancer.go [hunk 1]
```diff
@@ -0,0 +1,478 @@
...
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+
+			"l7_flavor_id": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+
+			"name": {
```
Comment (ShiChangkuo) [inline huaweicloud/resource_huaweicloud_elb_loadbalancer.go:147]: this parameter is **required** in the schame while it is optional in docs.

File: huaweicloud/resource_huaweicloud_elb_loadbalancer.go [hunk 2]
```diff
@@ -0,0 +1,478 @@
...
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils"
+)
+
+func ResourceLoadBalancerV3() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceLoadBalancerV3Create,
+		Read:   resourceLoadBalancerV3Read,
+		Update: resourceLoadBalancerV3Update,
+		Delete: resourceLoadBalancerV3Delete,
```
Comment (ShiChangkuo) [inline huaweicloud/resource_huaweicloud_elb_loadbalancer.go:23]: can the loadbalancer be imported?

File: docs/resources/elb_loadbalancer.md [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+
+* `iptype` - (Optional, String, ForceNew) Elastic IP type. Changing this parameter will create a new resource.
+
+* `bandwidth_charge_mode` - (Optional, String, ForceNew) Bandwidth billing type. Changing this parameter will create a new resource.
+
+* `sharetype` - (Optional, String, ForceNew) Bandwidth sharing type. Changing this parameter will create a new resource.
+
+* `bandwidth_size` - (Optional, Int, ForceNew) Bandwidth size. Changing this parameter will create a new resource.
+
+* `l4_flavor_id` - (Optional, String) The L4 flavor id of the load balancer.
```
Comment (ShiChangkuo) [inline docs/resources/elb_loadbalancer.md:128]: how users get the value of flavor id? can we provide a data source?

Comment (niuzhenguo) [reply docs/resources/elb_loadbalancer.md:128]: will provide flavor data source in the following up PR

File: huaweicloud/resource_huaweicloud_elb_loadbalancer.go (hunk 2 above)
Comment (niuzhenguo) [reply huaweicloud/resource_huaweicloud_elb_loadbalancer.go:23]: some attributes can not be fetched from GET, so will not provide import currently.

File: huaweicloud/resource_huaweicloud_elb_loadbalancer.go (hunk 1 above)
Comment (niuzhenguo) [reply huaweicloud/resource_huaweicloud_elb_loadbalancer.go:147]: will update in the following up patch set.

**Pull request merged**